docs/v1/P4Runtime-Spec.adoc

```diff
@@ -2446,26 +2285,26 @@ range.
 
 In the P4Runtime API version 1.0 (including minor version revisions), values
 of table key fields, action parameters, and fields in packet-in and packet-out
-headers between a device and the controller (see [#sec-controller-packet-meta]),
+headers between a device and the controller (see <<#sec-controller-packet-meta>>),
 may not be of type `int<W>`. The rules for encoding signed values thus only
-apply to messages of type `P4Data` (see [#sec-p4data-in-p4runtime-proto]).
+apply to messages of type `P4Data` (see <<#sec-p4data-in-p4runtime-proto>>).
 
 For a value of type `bit<W>`, the fewest number of bits required to represent
-the integer value $V > 0$ is the smallest integer $A$ such that $V \leq 2^A -
-1$.
+the integer value latexmath:[$V > 0$] is the smallest integer latexmath:[$A$] such that latexmath:[$V \leq 2^A - 1$].
 
 For a value of type `int<W>`, the fewest number of bits required to represent
-the integer value $V \neq 0$ in 2's complement form is the smallest integer $A$
-such that $-2^{A-1} \leq V \leq 2^{A-1} - 1$.
+the integer value latexmath:[$V \neq 0$] in "2's" complement form is the smallest integer latexmath:[$A$]
```

Comment:
I think it would be better to remove the quotes you added around "2's", unless that messes up the PDF output somehow.  I don't think it does mess anything up to leave them out.